index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./style.css">
+  <title>Hello Microverse</title>
+</head>
+<body>
+  <h2>Hello Microverse</h2>
+</body>
```

Comment:
- [ ] Please make sure that the text inside your h2 tag looks exactly the same as the one mentioned in the requirements.

![image](https://user-images.githubusercontent.com/74744787/157318382-0cb331ab-0e71-46e0-9e19-424d0b6eb0a5.png)
![image](https://user-images.githubusercontent.com/74744787/157318464-6139cc15-4993-4d8e-8886-e8c45d5b7ad4.png)